controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -2020,12 +2020,12 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, uint32_t max_res
         return -1;
     }
 
-    BML_NEIGHBOR_AP results[max_results_size] = {0};
+    BML_NEIGHBOR_AP results[max_results_size];
 
     uint8_t status             = 0;
     unsigned int results_count = max_results_size;
     int ret                    = bml_get_dcs_scan_results(ctx, radio_mac.c_str(),
-                                       reinterpret_cast<BML_NEIGHBOR_AP **>(&results),
+                                       reinterpret_cast<BML_NEIGHBOR_AP *>(results),
```

Comment:
Changed to `static_cast`, since the type of `results` is `BML_NEIGHBOR_AP[max_results_size]` while the requested type is `BML_NEIGHBOR_AP*`. And while the two are identical, I believe it is better to cast the type ourselves

Reply:
No, the two really are identical. Casting would give the wrong impression.

Reply:
Let me qualify that... By adding an explicit static cast, you hide a possible problem where the type of either `bml_get_dcs_scan_results` or `results` changes. True, the number of casts that are allowed with static_cast is *very* limited, but there are still a few possible, and it is very unlikely that they are what we want there. So removing the static cast may avoid bugs in the future (or rather, make sure they are captured by the compiler).

Reply:
Removed the cast all together
It builds with no issues, before when we used `BML_NEIGHBOR_AP **` without a cast the build failed, but when we changed the API call to `BML_NEIGHBOR_AP *` we don't need an explicit cast